code/modules/clothing/gloves/miscellaneous.dm

```diff
@@ -183,3 +183,153 @@
 	name = "CentCom intern envirogloves"
 	icon_state = "internplasma"
 	inhand_icon_state = "internplasma"
+
+//Nemesis Solutions Gloves
+
+/obj/item/clothing/gloves/rapid/nemesis
+	name = "rapid stun gloves"
+	desc = "A pair of high-tech gloves with \"Nemesis Solutions\" written on the inside."
+	icon_state = "nemesis"
+	inhand_icon_state = "black"
+	transfer_prints = FALSE
+
+	var/charge = 0
+	var/datum/martial_art/nemesis/style
+	//To prevent spam from overcharge warnings
+	var/antispam = 0
+
+	var/obj/item/shield/energy/nemesis/shield
+
+/obj/item/clothing/gloves/rapid/nemesis/Initialize()
+	. = ..()
+	var/datum/component/wearertargeting/punchcooldown/punch_cooldown = GetComponent(/datum/component/wearertargeting/punchcooldown)
+	punch_cooldown.warcry = null
+	punch_cooldown.UnregisterSignal(src, COMSIG_ITEM_ATTACK_SELF)
+	style = new()
+	shield = new(src)
+
+/obj/item/clothing/gloves/rapid/nemesis/equipped(mob/user, slot)
+	. = ..()
+
+	if(!ishuman(user))
+		return
+
+	if(slot == ITEM_SLOT_GLOVES)
+		var/mob/living/student = user
+		style.teach(student, 1)
+		ADD_TRAIT(student, TRAIT_NOGUNS, NEMESIS_TRAIT)
+
+/obj/item/clothing/gloves/rapid/nemesis/dropped(mob/user)
+	. = ..()
+
+	if(!ishuman(user))
+		return
+
+	var/mob/living/owner = user
+	style.remove(owner)
```

Comment:
okay, but what if a wizard mind transfers into someone wearing these gloves, then takes off the gloves? will the person in the wizard's body retain their martial arts knowledge?